### PR TITLE
Changed private to public for Angular

### PR DESCRIPTION
**Pull request by erichunsinger**

### CODE REVIEW

**Comment by erichunsinger** [comment]
changed service variables from public to private while leaving the service methods public

File: src/app/Job-Service/job.service.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import Profile from '../models/Profile';
 })
 export class JobService {
 
-  private url = 'http://localhost:8765/interview-service/jobs';
+  public url = 'http://localhost:8765/interview-service/jobs';
```
Comment (IkenoXamos) [inline src/app/Job-Service/job.service.ts:11]: Do the member variables of a service need to be public?

File: src/app/profile.service.ts [hunk 2]
```diff
@@ -10,9 +10,9 @@ import { Observable } from 'rxjs';
 })
 export class ProfileService {
 
-    private url1 = 'http://localhost:8765/interview-service/profiles/allProfiles';
+    public url1 = 'http://localhost:8765/interview-service/profiles/allProfiles';
```
Comment (IkenoXamos) [inline src/app/profile.service.ts:13]: Do the member variables of a service need to be public?

File: src/app/skill.service.ts [hunk 3]
```diff
@@ -13,8 +13,8 @@ export class SkillService {
   private profileSkills = new BehaviorSubject<Skill>(null);
   profileSkills$ = this.profileSkills.asObservable();
 
-  private url1 = 'http://localhost:8765/interview-service/skills/allSkills';
-  private url2 = 'http://localhost:8765/interview-service/skills/insertSkill';
+  public url1 = 'http://localhost:8765/interview-service/skills/allSkills';
```
Comment (IkenoXamos) [inline src/app/skill.service.ts:16]: Do the member variables of a service need to be public?

**Review by IkenoXamos — CHANGES_REQUESTED** [review]
As far as I can tell the member variables of a service should remain private, and only the member variables of a component should be public. This is in order for that field to be accessible for databinding. Since services don't have a template, there is no need for any databinding to occur for a service's member variables.

**Pull request closed without merging**